dev-tools/test.sh

```diff
@@ -6,17 +6,28 @@
 # shellcheck source=./dev-tools/_functions.sh
 source "$(dirname "${BASH_SOURCE[0]}")/_functions.sh"
 
-# Delete outdated code coverage
-rm -rf "${BASE_DIR:?}/htmlcov/"
+# Delete outdated code coverage report
+CODE_COVERAGE_DIR="${BASE_DIR:?}/htmlcov/"
+rm -rf "${CODE_COVERAGE_DIR}"
 
 require_installed
 require_database
 
 # Set dummy key to enable SUMM.AI during testing
 export INTEGREAT_CMS_SUMM_AI_API_KEY="dummy"
 
-deescalate_privileges pipenv run pytest --disable-warnings --quiet --numprocesses=auto --cov=integreat_cms --cov-report html
+# If only particular tests should be run, test path can be passed as CLI argument
+TEST_PATH=$1;
+
+if [[ -z "$TEST_PATH" ]]; then
+    deescalate_privileges pipenv run pytest --disable-warnings --quiet --numprocesses=auto --cov=integreat_cms --cov-report html
+else
+    deescalate_privileges pipenv run pytest "$TEST_PATH" --disable-warnings --quiet --numprocesses=auto
+fi
+
 echo "✔ Tests successfully completed " | print_success
 
-echo -e "Open the following file in your browser to view the test coverage:\n" | print_info
-echo -e "\tfile://${BASE_DIR}/htmlcov/index.html\n" | print_bold
+if [ -d "$CODE_COVERAGE_DIR" ]; then
+    echo -e "Open the following file in your browser to view the test coverage:\n" | print_info
+    echo -e "\tfile://${CODE_COVERAGE_DIR}/index.html\n" | print_bold
```

Comment:
Minor: at least in my case, the path looks like `file:///home/[...]/htmlcov//index.html` (with two slashes at the end). IDK if that happens on all platforms though.

Reply:
Ah, it's my mistake that I didn't notice. Thanks